app/boards/shields/jorne/jorne.conf

```diff
@@ -0,0 +1,6 @@
+# Uncomment the following lines to enable the Jorne RGB Underglow
+# ZMK_RGB_UNDERGLOW=y
```

Comment:
Last thing, I think this got last in your last round of changes!

Reply:
It was probably the one before that. Kinda got lost amidst all the rebasing...